api/src/routers/meals.js

```diff
@@ -0,0 +1,93 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const mealRouter = express.Router();
+// get all meal
+mealRouter.get("/", async (req, res, next) => {
+    try{
+        const all_meals = await knex("meal").select("*");
```

Comment:
`select("*")` is there default, so `knex("meal")` will do

Reply:
I have made changes now